tgui/packages/tgui/interfaces/ai_fixer.js

```diff
@@ -0,0 +1,94 @@
+import { useBackend } from "../backend";
+import { Button, ProgressBar, Box, LabeledList } from "../components";
+import { Window } from "../layouts";
+
+export const AIFixer = (props, context) => {
+  const { act, data } = useBackend(context);
+  const laws = data.laws || [];
+  if (data.occupant == null) {
+    return (
+      <Window>
+        <Window.Content>
+          <Box>
+            <h3>No artificial intelligence detected.</h3>
+          </Box>
+        </Window.Content>
+      </Window>
+    );
+  } else {
+    let workingAI = null;
+    if (data.stat == 2 || data.stat == null) {
+      workingAI = false;
+    } else { workingAI = true; }
+
+    let integrityColor = null;
+    if (data.integrity >= 75) {integrityColor = 'green';}
+    else if (data.integrity >= 25) {integrityColor = 'yellow';}
+    else {integrityColor = 'red';}
+
```

Comment:
Maybe add code here that defines let aiLawBlock; and then loops through lawsList to add stuff to it.
Then maybe use {aiLawBlock} below to include it. That way you're not trying to calculate it in the return() section.